video/quickstart.rb

```diff
@@ -0,0 +1,53 @@
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START videointelligence_quickstart]
+require "google/cloud/video_intelligence/v1beta1"
+
+video_client = Google::Cloud::VideoIntelligence::V1beta1::VideoIntelligenceServiceClient.new
+features     = [:LABEL_DETECTION]
+path         = "gs://demomaker/cat.mp4"
+
+# Register a callback during the method call
+operation = video_client.annotate_video path, features do |operation|
+  raise operation.results.message? if operation.error?
+  puts "Finished Processing."
+
+  # first result is retrieved because a single video was processed
+  annotation_result = operation.results.annotation_results.first
+
+  annotation_result.label_annotations.each do |label_annotation|
+    puts "Label description: #{label_annotation.description}"
+    puts "Locations:"
+
+    label_annotation.locations.each do |location|
+      positions = "Entire video"
+
+      if location.level != :VIDEO_LEVEL
```

Comment:
Also, if an offset is seconds.... how about... instead of `#{offset}s` ... we do `#{start_seconds}` and put the word 'seconds' in the variable name